Leanplum-SDK/Classes/Managers/LPAppIconManager.m

```diff
@@ -72,7 +72,9 @@ + (void)uploadAppIconsOnDevMode
     [request onError:^(NSError *error) {
         LPLog(LPError, @"Fail to upload app icons: %@", error.localizedDescription);
     }];
-    [[LPRequestSender sharedInstance] sendNow:request withDatas:requestDatas];
+    request.datas = requestDatas;
+    request.requestType = Immediate;
```

Comment:
Move `requestType` to be as part of the builder pattern instead setting it like this

Reply:
I agree, it will make it more clear if the request will be batched or send immediately.